pelican/server.py

```diff
@@ -14,6 +16,27 @@
 from six.moves import socketserver
 
 
+class ServerCommand:
+    def __init__(self):
+        parser = argparse.ArgumentParser(
+            description='Pelican Development Server',
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter
+        )
+        parser.add_argument("port", default=8000, type=int, nargs="?",
+                            help="Port to Listen On")
+        parser.add_argument("server", default="", nargs="?",
+                            help="Interface to Listen On")
+        parser.add_argument('--ssl', action="store_true",
+                            help='Activate SSL listener')
+        parser.add_argument('--cert', default="./cert.pem", nargs="?",
+                            help='Path to certificate file. ' +
+                            'Relative to current directory')
+        parser.add_argument('--key', default="./key.pem", nargs="?",
+                            help='Path to certificate key file. ' +
+                            'Relative to current directory')
+        self.options = parser.parse_args()
+
```

Comment:
Why is this a class? If you wanted to leave it out of `__main__` block for clarity sake, you could just make it a function that returns `parser.parse_args()`.

Reply:
Like [this](https://github.com/getpelican/pelican/blob/master/pelican/__init__.py#L262) and [usage](https://github.com/getpelican/pelican/blob/master/pelican/__init__.py#L395)

Reply:
@avaris not a problem i will update